test/test.js

```diff
@@ -0,0 +1,15 @@
+/* global describe, it, assert */
```

Comment:
Could we use eslint-env mocha here? https://eslint.org/docs/user-guide/configuring#specifying-environments